index.js

```diff
@@ -68,3 +69,16 @@ function sortByLast(a, b) {
   }
   return 0
 }
+
+function search() {
+  $("#search").on("keyup", function(event) {
+    var text = $("#search").val().toLowerCase()
+    $(".name").each(function(index) {
+      if ($(this).text().toLowerCase().includes(text)) {
+        $(this).closest("div").show()
+      } else {
+        $(this).closest("div").hide()
+      }
+    })
+  })
+}
```

Comment:
Overall, this is an interesting approach to search (and valid). There are at least two ways to go about this:
- have an immutable list of names and filter the view based on the search query (your approach)
- update the list (or a version of the list) based on the search query and have `listContacts` reflect that change to the data.

It might be interesting to explore the other option and see what that would look like. Again both approaches are fine. It just might be an interesting thought experiment to interact with.